src/pages/user_profile/ResearcherStatus.jsx

```diff
@@ -77,7 +77,7 @@ export default function ResearcherStatus(props) {
       <a href={accountLink}>
         {accountLabel} Account
       </a>
-      &nbsp;is required to submit a dar.
+      &nbsp;is required to submit a data access request.
```

Comment:
```suggestion
      &nbsp;is required to submit a Data Access Request (DAR).
```

Perhaps worth uppercasing? We essentially use this as a proper noun in a few other places, so this could provide clarity.